### PR TITLE
chore: update actions config 

### PR DESCRIPTION
**Pull request by thonatos**
> changes can be found in commit list.  😉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # bug-versions
 
 [![NPM version][npm-image]][npm-url]
-[![build status][travis-image]][travis-url]
 [![Known Vulnerabilities][snyk-image]][snyk-url]
 [![npm download][download-image]][download-url]
+[![build status][travis-image]][travis-url]
```
Comment (atian25) [inline README.md:6]: travis 干掉

File: package.json [hunk 2]
```diff
@@ -527,12 +532,19 @@
...
     "@semantic-release/changelog": "^3.0.2",
     "egg-ci": "^1.15.0",
+    "husky": "^4.2.5",
     "semantic-release": "^15.13.21"
   },
   "ci": {
-    "type": "github"
+    "type": "github",
+    "os": {
+      "github": "linux"
```
Comment (atian25) [inline package.json:545]: 不是 ubuntu 么

Comment (thonatos) [reply package.json:545]: 这里 linux 就是 ubuntu，actions 的配置

File: README.md (hunk 1 above)
Comment (thonatos) [reply README.md]: done.

File: package.json (hunk 2 above)
Comment (thonatos) [reply package.json:545]: https://github.com/eggjs/egg-ci/blob/44f207c716f7d3c5b37357e2afb497d01db03809/install.js#L71

**Pull request merged**